dropwizard-validation/src/test/java/io/dropwizard/validation/SelfValidationTest.java

```diff
@@ -55,10 +58,12 @@ protected void append(ILoggingEvent iLoggingEvent) {
     @BeforeAll
     static void setUp() {
         LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
```

Comment:
We don't need the `LoggerContext` here. As you already did in the `messageParametersExample()` method, we can get Logback's `Logger` object by casting the logger obtained from the `LoggerFactory.getLogger(...)` call.

Reply:
Right you are, it's gone.